src/main/java/com/conveyal/r5/analyst/cluster/AnalystWorker.java

```diff
@@ -825,6 +821,20 @@ private void logQueueStatus() {
         LOG.debug("Waiting tasks: high priority: {}, batch: {}", highPriorityExecutor.getQueue().size(), batchExecutor.getQueue().size());
     }
 
+    public static boolean isHighPriorityTask (GenericClusterRequest req) {
```

Comment:
Conditional-based polymorphism looks a bit suspicious to me. Couldn't this be handled with an isHighPriority method on GenericClusterRequest?